.github/workflows/ci-spell-checking.yml

```diff
@@ -0,0 +1,44 @@
+name: ci-spell-checking
+
+on:
+  push:
+    branches: 
+    - main
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+    - ready_for_review
+    branches: 
+    - main
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

Comment:
why is this needed for spell check?

Reply:
I was using for other thing. I will remove it.